lib/action.js

```diff
@@ -648,17 +677,43 @@ function action(system) {
 			}
 		}
 		self.system.emit('actions_update');
-		for (var n in self.release_actions) {
-			var x = n.split(/:/);
-			if (x[0] == id) {
-				delete self.release_actions[n];
+	});
+
+	system.on('action_subscribe_bank', function (page, bank) {
+		if (self.bank_actions[page] !== undefined && self.bank_actions[page][bank] !== undefined) {
+			// find all instance-ids in feedbacks for bank
+			for (var i in self.bank_actions[page][bank]) {
+				self.subscribeAction(self.bank_actions[page][bank][i]);
+			}
+		}
+		if (self.bank_release_actions[page] !== undefined && self.bank_release_actions[page][bank] !== undefined) {
+			// find all instance-ids in feedbacks for bank
+			for (var i in self.bank_release_actions[page][bank]) {
+				self.subscribeAction(self.bank_release_actions[page][bank][i]);
 			}
 		}
-		self.system.emit('release_actions_update');
 	});
 
+	system.on('action_unsubscribe_bank', function (page, bank) {
+		if (self.bank_actions[page] !== undefined && self.bank_actions[page][bank] !== undefined) {
+			// find all instance-ids in feedbacks for bank
```

Comment:
feedbacks? :P

Reply:
Not at all obvious I copied this from the implementation done for feedback :P